pkg/kv/store.go

```diff
@@ -20,6 +23,14 @@ var (
 	ErrUnknownDriver       = errors.New("unknown driver")
 )
 
+func NewPath(p ...string) string {
+	const minPathLength = 2
+	if len(p) < minPathLength {
+		panic("invalid path length")
+	}
```

Comment:
Why? keep it simple and just enable us to build a single word path - which will not include any sep.